kasa/cli.py

```diff
@@ -396,11 +397,19 @@ def _nop_echo(*args, **kwargs):
             connection_type=ctype,
         )
         dev = await Device.connect(config=config)
+        device_updated = True
     else:
-        echo(
-            "No --type or --device-family and --encrypt-type defined, "
-            + f"discovering for {discovery_timeout} seconds.."
-        )
+        if device_family or encrypt_type:
```

Comment:
Considering we have a generic support for all `SMART.*` based on the negotiation, I'm wondering if we should just fallback to use some generic device family if encrypt type is set?

If not, I think we can merge this as is, thanks!

Reply:
We could do that although I think the documentation needs to be updated to avoid issues with devices that won't work with the default (HS100(UK)) and to explain these parameters better in general. Issue https://github.com/python-kasa/python-kasa/issues/991 opened to track this.